.github/workflows/spack.yml

```diff
@@ -23,8 +23,8 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/.spack-cache
-          key: cache-${{ github.sha }}
-          restore-keys: cache-
+          key: ccache-${{ github.run_id }}
```

Comment:
no reason.

Reply:
Interestingly it does matter. The commit where I removed it failed, adding back it passed. I can think of no other reason that that the action in fact uses `ccache`, because what else would you use :)

Reply:
That's very obtuse. Please make a big, red *flashing sign* for that.

Reply:
Dug a little, and could not find any reason to believe it actually matters. Probably the cache was just cold after deleting a `c`. I changed it to sth that assumes no magic, and trust that it'll work. It'll take a run or two extra if you want to be absolutely sure.

Reply:
There's no need for the double "c" :)